docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -18,5 +18,153 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+`SHOW TABLE EXTENDED` will show information for all tables matching the given regular expression.
+Output includes basic table information and file system information like `Last Access`, 
+`Created By`, `Type`, `Provider`, `Table Properties`, `Location`, `Serde Library`, `InputFormat`, 
+`OutputFormat`, `Storage Properties`, `Partition Provider`, `Partition Columns` and `Schema`.
+
+Users cannot use regular expression for the table name if a partition specification is present. It will output 
```

Comment:
Last nit: _a_ regular expression. But how about:
"If a partition specification is present, it outputs the given partition's ... Note that a table regex cannot be used with a partition specification."

Reply:
@srowen 
If the PARTITION key present in the query, the pattern mentioned in LIKE will not be treated as a regex. It will be treated as normal name.

for example. **If we mention the regex with LIKE and PARTITION keyword, it fails like below.**
![image](https://user-images.githubusercontent.com/7912929/67885787-3410c780-fb6e-11e9-9a57-012d8a946670.png)

**But if mention the full name like below it works**
![image](https://user-images.githubusercontent.com/7912929/67885883-63273900-fb6e-11e9-97e2-2ea705ff0b8a.png)

Is it better to mention this exception scenario in the examples section?

Reply:
Yes, I think that agrees with the text I suggested? there's no problem documenting this in the docs here.

Reply:
@srowen 
Documented Exception scenario.

Reply:
Yes, but I don't think you changed this sentence. I'm suggesting a typo fix and rewrite right here.

Reply:
@shivusondur still at least one more change here.

Reply:
@srowen 
Sorry for late, I updated it according to your suggestion.